x-pack/plugins/apm/public/components/app/ServiceMap/Popover/Contents.tsx

```diff
@@ -116,6 +116,14 @@ const ANOMALY_DETECTION_DISABLED_TEXT = i18n.translate(
   }
 );
 
+const ANOMALY_DETECTION_NO_DATA_TEXT = i18n.translate(
+  'xpack.apm.serviceMap.anomalyDetectionPopoverNoData',
+  {
+    defaultMessage:
+      'No anomaly score found with the current filters. See details in the anomaly explorer:',
```

Comment:
```suggestion
      'We couldn't find any anomaly score within the current time range. See details in the anomaly explorer.',
```

@ogupte do you think this copy covers the scenario? I'm just trying to make a little more assuring while still being descriptive about they might be looking at the wrong time range or there's something wrong with the data feed. Also since we're currently not taking environment filter into account, we best just clarify that it's simply the time range that we're looking at.

Reply:
Is it only possible to have one anomaly score per service in a given time frame? If so, what about `an` instead of `any`?

```
We couldn't find an anomaly score within the current time range. See details in the anomaly explorer.
```

Reply:
We display the max anomaly score within the time range here, so just 1 score. If any service has more than 1 ml jobs associated with it, it will only display the one with the higher score.

Reply:
Neat. I think this works then: "We couldn't find an anomaly score within the selected time range. See details in the anomaly explorer."